### PR TITLE
Update pricing.yml

### PR DESCRIPTION
**Pull request by mheadd**
Add search.gov and DAP list of items and pricing page.

Fixes issue(s) #493  .

Proposed changes in this pull request:
- Added search.gov and DAP to list of items on pricing page that come bundle with annual platform access fee.

/cc @davemcorwin 

[CircleCI](https://circleci.com/gh/18F/federalist.18f.gov/tree/mheadd-patch-1)

[:sunglasses: PREVIEW](https://federalist-a44a7672-afb1-473e-acb2-067f644c0967.app.cloud.gov/preview/18f/federalist.18f.gov/mheadd-patch-1/pricing/)

### CODE REVIEW

**Review by davemcorwin — COMMENTED** [review]
@mheadd I think we touched base about this previously, but my thought is that we shouldn't add these because they are TTS services available to anyone for free. @apburnes thoughts? We could also run it by Ron or #cg-business.

**Comment by apburnes** [comment]
We have a page for search.gov and only mention DAP in passing. These are two services our customers get a lot of mileage out of so it maybe worth stating it more clearly and upfront.  Maybe something like (but more elegantly) `U.S Web Design System starter kits integrated with Search.gov and GSA Digital Analytics Program to get your site running within minutes`

**Comment by mheadd** [comment]
@apburnes Agreed. I can make that change using the text you suggest.

**Comment by mheadd** [comment]
Incorporated @apburnes' changes. This look OK to you both? cc: @davemcorwin

**Comment by apburnes** [comment]
I would suggest putting the new added line right after the first USWDS bullet just to keep things grouped. But looks 💯 to me.

**Comment by mheadd** [comment]
Move that bullet and made a small mod to the text, mostly to remove the duplicate `get your site running within minutes` phrase.

File: _data/pricing.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ plans:
   - title: "Platform Access to Federalist $24,949/year"
     features:
       - { text: U.S Web Design System starter kits in Javascript and Ruby to get your site running within minutes. }
+      - { text: USWDS starter kits integrated with Search.gov and GSA Digital Analytics Program to further enhance your site. }
       - { text: GSA-maintained GSA Authority to Operate (ATO) for all your sites. }
       - { text: Automated tests and previews before going live. }
```
Comment (davemcorwin) [inline _data/pricing.yml:7]:
```suggestion
      - { text: Automated previews before going live. }
```

File: _data/pricing.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ plans:
   - title: "Platform Access to Federalist $24,949/year"
     features:
       - { text: U.S Web Design System starter kits in Javascript and Ruby to get your site running within minutes. }
+      - { text: USWDS starter kits integrated with Search.gov and GSA Digital Analytics Program to further enhance your site. }
```
Comment (davemcorwin) [inline _data/pricing.yml:5]:
```suggestion
      - { text: U.S. Web Design System Starter kits in Javascript and Ruby with built-in integrations with Search.gov and GSA Digital Analytics Program to get your site running within minutes. }
```

**Review by davemcorwin — APPROVED** [review]
Seemed a bit repetitive so I tried to consolidate the 2 items. Looks 💯

**Pull request merged**